x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/support/WatcherIndexTemplateRegistry.java

```diff
@@ -46,17 +53,23 @@
             TEMPLATE_CONFIG_TRIGGERED_WATCHES, TEMPLATE_CONFIG_WATCH_HISTORY, TEMPLATE_CONFIG_WATCHES
     };
 
+    public static final PolicyConfig POLICY_WATCH_HISTORY = new PolicyConfig("watch-history-policy", "/watch-history-policy.json");
```

Comment:
I remember the discussion to _not_ use a `.` in the name since this is something that we don't mind users changing. However, I wonder if in the UI there should be some kind of differentiation policies we add vs. policies they add.  If so we may want the `.` after all. 

@bmcconaghy - Do you see any reason that the UI for ILM (now or in the future) should differentiate between policies added by the system vs. policies added by a human ?

Reply:
> I remember the discussion to _not_ use a `.` in the name since this is something that we don't mind users changing. However, I wonder if in the UI there should be some kind of differentiation policies we add vs. policies they add. If so we may want the `.` after all.
> 
> @bmcconaghy - Do you see any reason that the UI for ILM (now or in the future) should differentiate between policies added by the system vs. policies added by a human ?

When we're making amendments to config through the API, we expect system-created objects to being with a `.` character, and hence we avoid those.

I think it would make sense to adhere to that pattern; it's a bit surprising that you've deviated from it.

Moreover, if a user were to delete this policy because they thought it had been created by another user rather than the system, the system would consequently behave differently to how it's specified to behave in the docs.